From f07711a61fc9ff2be82fe338853ef1ae4a3d669d Mon Sep 17 00:00:00 2001
From: Guillaume Tucker <guillaume.tucker@collabora.com>
Date: Tue, 11 May 2021 16:21:11 +0100
Subject: [PATCH 2/3] CHROMEOS Change BASE_PATH for
 storage.chromeos.kernelci.org

Change the hard-coded BASE_PATH to point to the location where the
storage.chromeos.kernelci.org files are stored.

Signed-off-by: Guillaume Tucker <guillaume.tucker@collabora.com>
---
 app/utils/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/app/utils/__init__.py b/app/utils/__init__.py
index 081570673f30..febb45d955e6 100644
--- a/app/utils/__init__.py
+++ b/app/utils/__init__.py
@@ -28,7 +28,7 @@ import re
 import models
 import utils.log
 
-BASE_PATH = "/var/www/images/kernel-ci"
+BASE_PATH = "/data/storage.chromeos.kernelci.org"
 LOG = utils.log.get_log()
 
 # Build log file names.
-- 
2.20.1

